server/heroku/heroku.go

```diff
@@ -167,3 +167,7 @@ func UserFromContext(ctx context.Context) *empire.User {
 	}
 	return u
 }
+
+func findMessage(r *http.Request) (string, error) {
```

Comment:
i went with `(string, error)` return value here to support a flag where all commit messages could be required in the future.